### ISSUE TITLE
DAO-less build

### ISSUE THREAD

**Issue by ghost** [bisq-network/bisq#4862]
### Description

Build a version without the DAO. I'm getting 300+% CPU usage and 5GB of RAM used and all it seems to be doing is syncing the DAO. I use bisq for BTC. I literally cannot click a button or use bisq atm. 

Went on for like 30 minutes, went away as soon as it was done syncing. 

#### Version

1.5.0

### Steps to reproduce

Sync DAO, try to use bisq

### Expected behaviour

Can use bisq

### Actual behaviour

Cannot use bisq

#### Device or machine

Macbook Pro 10.15.5

**Comment by huey735** [comment]
Run `Bisq --daoActivated=false`
```
  --daoActivated=<Boolean> (default: true)
        Developer flag. If true it enables dao phase 2 features.
```

**Comment by p3yot3** [comment]
> Run `Bisq --daoActivated=false`
> 
> ```
>   --daoActivated=<Boolean> (default: true)
>         Developer flag. If true it enables dao phase 2 features.
> ```

Can this be used on all OS's?

**Comment by huey735** [comment]
@p3yot3 yes

**Comment by ghost** [comment]
Should be a config option in the settings or asked on first launch.

**Comment by huey735** [comment]
@AnthonyRonning consider proposing it on the [Proposals Repository](github.com/bisq-network/proposals/issues).

**Comment by ghost** [comment]
Doesn't look like something I have access to.

**Comment by p3yot3** [comment]
> > Run `Bisq --daoActivated=false`
> > ```
> >   --daoActivated=<Boolean> (default: true)
> >         Developer flag. If true it enables dao phase 2 features.
> > ```
> 
> Can this be used on all OS's?

Doesn't work with windoze - Bisq doesn't connect when using this command.

**Comment by huey735** [comment]
@p3yot3 this is how to use the command line for Windows: https://bisq.wiki/Running_from_the_command_line

**Comment by M0ns1gn0r** [comment]
I started it without DAO, the result turned out to be even worse than with DAO. The UI becomes unresponsive for ~90 seconds, then it is responsive for ~5 seconds, then unresponsive again (these cycles don't end).

The client is unusable in this mode.


P.S. maybe this would tell you anyting.
```
mar.-13 14:15:18.282 [JavaFX Application Thread] INFO  b.n.p.s.P2PDataStorage: Processing 1044 persistableNetworkPayloads took 95550 ms. 
mar.-13 14:15:18.282 [JavaFX Application Thread] INFO  b.n.p.p.g.RequestDataHandler: Processing GetDataResponse took 95552 ms 
```

**Comment by cd2357** [comment]
> I'm getting 300+% CPU usage and 5GB of RAM used and all it seems to be doing is syncing the DAO. I use bisq for BTC. I literally cannot click a button or use bisq atm.

Should be fixed in the latest version (v1.6.4). It upgrades several frameworks and libraries that improve overall performance and reduce resource consumption (especially RAM).

Please give it a try and let us know if this is still a problem for you.
